lang/russian.json

```diff
@@ -1,146 +1,146 @@
-    "borderWidth": "Ширина границы",
+    "borderWidth": "Толщина границы",
     "brightnessTool": "Яркость",
     "browseFiles": "Просмотр файлов",
-    "buttonsBlockAlignment": "Редактирование выравнивания",
-    "buttonsBlockGap": "Промежуток между кнопками",
-    "buttonsBlockPlaceholder": "Вводи текст ...",
+    "buttonsBlockAlignment": "Редактировать выравнивание",
+    "buttonsBlockGap": "Расстояние между кнопками",
+    "buttonsBlockPlaceholder": "Введите текст ...",
     "buttonsBlockSettings": "Настройки",
     "buttonsBlockTitle": "Кнопки",
-    "cacheClearedSuccess": "Кэш успешно очистился!",
+    "cacheClearedSuccess": "Кэш был успешно очищен!",
     "cacheDisabled": "Кэширование отключено",
     "cacheEnabled": "Кэширование включено",
-    "cachePurgedError": "Не удалось очистить каталог кэша!",
-    "cachePurgedSuccess": "Каталог кэша был успешно очищен!",
-    "cancel": "Отмена",
-    "caption": "Надпись",
+    "cachePurgedError": "Не удалось очистить папку кэша!",
+    "cachePurgedSuccess": "Папка кэша была успешно очищена!",
+    "cancel": "Отменить",
+    "caption": "Подпись",
     "changesLoseConfirmation": "Все изменения будут потеряны",
-    "changesLoseConfirmationHint": "Ты уверен, что хочешь продолжить?",
-    "cinemascope": "Cinemascope",
-    "className": "Класс CSS",
-    "classicTv": "Классическое телевидение",
+    "changesLoseConfirmationHint": "Вы уверены, что хотите продолжить?",
+    "cinemascope": "Синемаскоп",
+    "className": "Имя CSS класса",
+    "classicTv": "Классический TV",
     "clickToDelete": "Нажмите, чтобы удалить",
     "close": "Закрыть",
     "code": "Исходный код",
     "color": "Цвет",
-    "confirmCancelUpload": "Хочешь отменить загрузку?",
-    "confirmDeletePage": "Хочешь удалить текущую страницу?",
-    "confirmDeleteSelectedFiles": "Хочешь удалить выбранные файлы?",
-    "confirmDiscardUnsaved": "Ты уверен, что хочешь отменить несохраненные изменения?",
+    "confirmCancelUpload": "Вы хотите отменить загрузку?",
+    "confirmDeletePage": "Вы хотите удалить текущую страницу?",
+    "confirmDeleteSelectedFiles": "Вы хотите удалить выбранные файлы?",
+    "confirmDiscardUnsaved": "Вы уверены, что хотите отменить свои несохраненные изменения?",
     "continue": "Продолжить",
     "contrastTool": "Контраст",
-    "convertTo": "Конвертируй в",
-    "copyUrlClipboard": "Копирование URL в буфер обмена",
+    "convertTo": "Конвертировать в",
+    "copyUrlClipboard": "Копировать URL-адрес в буфер обмена",
     "cornerRadius": "Радиус угла",
     "couldNotSaveError": "Не удалось сохранить изменения",
-    "cropSizeLowerThanResizedWarning": "Обратите внимание, что выбранная область обрезки меньше, чем применяемый размер, что может привести к снижению качества.",
-    "cropTool": "Crop",
+    "cropSizeLowerThanResizedWarning": "Обратите внимание, что выбранная область после обрезки будет меньшего размера чем размеры изображений по умолчанию, что может привести к снижению качества изображения",
```

Comment:
I have the feeling the meaning of the sentence has changed. I am no expert, but all AI translations diverge a lot from the original text.